core/controllers/ingest.py

```diff
@@ -47,6 +47,9 @@ def ingest(body, excel_file):
     :raises ValidationError: If the data fails validation against the specified schema.
     """
     source_type = body.get("source_type")  # required
+    # TODO: Do we want to retain the old functionality? If so, how do we want to toggle it? By request param like this?
+    # user_centered_messaging = body.get("user_centered_messaging")
+    user_centered_messaging = True  # TODO: remove hard-coded variable
```

Comment:
Why would we need to toggle unless we plan to have two sets of messages (e.g. one for us, one for the users)? If the users are now responsible for their own uploads, then we shouldn't require specific messages for us (or else we're more like admins). Also the end result of the user-centred messages should still be sufficient for us to find & fix problems.

Reply:
I was also thinking this.. just wasn't sure yet so left this in for now.
> "# TODO: Do we want to retain the old functionality?"

There are other "Failures" that we collect and that we won't add "UC" versions of, although in our use case we never return them due to the data that is outputted from transformation so maybe it's a void point. And if there ever were some that we didn't have UC messages for that were returned then they almost certainly should have UC messages.

The crux of the decision is will those old style messages ever be more useful for us as developers. If that was the case though it probably suggests we haven't done this well - rather than having the toggle is useful.

My thinking is that we don't need it and I'll remove the toggle.

Reply:
The only use case in which the legacy messages would serve a purpose is if we're ingesting a dataset for which we don't have the original form (R1 & R2, for instance). Otherwise, provided we retain the value from which the error has arisen we should be able to locate the problem in the spreadsheet as easily as with the legacy messages (I think!).